src/model/applications/repository.ts

```diff
@@ -8,8 +8,8 @@ export type ContributionApplicationDto = {
   contributor_id: number;
 };
 
-export type ListFromContributionQueryParams = {
-  contributorId: number;
+export type ListQueryParams = {
```

Comment:
I would avoid having the notion of query params in the repository interface – I know it was here before but let's improve it –. Maybe simply `ListParams`? Maybe even having a dedicated type is overkill here

Reply:
updated. I set `ListParams` to stay consistent with `CreateParams`